api.md

```diff
@@ -3518,6 +3519,28 @@ SELECT timescaledb_post_restore();
 ```
 ---
 
+## timescaledb.license [](timescaledb_license)
+
+View or set currently used TimescaleDB license.
+
+It is possible to downgrade currently used license from `timescale` to `apache` by setting
+the `timescaledb.license` GUC in the server configuration file or using the
+server command line.
```

Comment:
I am not sure we should focus on downgrades here. I suggest adding a bit more nuance:

```suggestion
It is possible to limit access to features by license by changing the `timescaledb.license` settings parameter in the server configuration file or on the server command line. For instance, by setting `timescaledb.license`  to `apache`, it is only possible to use TimescaleDB features that are implemented under the Apache 2 license. The default value is `timescale`, however, which allows access to all features.
```